sdk/cognitivelanguage/azure-ai-language-questionanswering/README.md

```diff
@@ -129,6 +146,95 @@ output = await client.get_answers(
 )
 ```
 
+### QuestionAnsweringProjectsClient
+
+#### Create a new project
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.language.questionanswering.projects import QuestionAnsweringProjectsClient
+
+# get service secrets
+endpoint = os.environ["AZURE_QUESTIONANSWERING_ENDPOINT"]
+key = os.environ["AZURE_QUESTIONANSWERING_KEY"]
+
+# create client
+client = QuestionAnsweringProjectsClient(endpoint, AzureKeyCredential(key))
+with client:
+
+    # create project
+    project_name = "IssacNewton"
+    project = client.create_project(
+        project_name=project_name,
+        options={
+            "description": "biography of Sir Issac Newton",
+            "language": "en",
+            "multilingualResource": True,
+            "settings": {
+                "defaultAnswer": "no answer"
+            }
+        })
+
+    print("view created project info:")
+    print(u"\tname: {}".format(project["projectName"]))
```

Comment:
We shouldn't need the `u` prefix - this is a Python 2.7 format that we no longer support.